Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1769,7 +1769,8 @@ extension Purchases: InternalPurchasesType {
     }
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
-    func productEntitlementMapping() async throws -> ProductEntitlementMapping {
+    // swiftlint:disable:next missing_docs
+    @_spi(Internal) public func productEntitlementMapping() async throws -> ProductEntitlementMapping {
```

Comment:
Hmm I think some people might go the extra mile and opt in, in order to use this (there have been some requests before).

Just wondering, why do we need to expose this? Can we mock it in some way if we need it for the app?